Automattic/simplenote-android
# Update Passcode Layout

### Fix
Update the passcode layout with the new design.  Note that the number pad in the tablet layout is slightly lower in the ***After*** screenshot compared to the ***Design*** mockup.  The number pad is not only centered in the layout, but also assigned below the "Enter your passcode" text to ensure there is no overlap between those views.  As a consequence, the number pad is slightly lower.  Also, the error message location was changed from the top of the screen to the default location of `Toast` components for consistency.  All references to "PIN" were changed to "passcode" in the interface.  The `colorPrimary` attribute was added to the `Theme.Simplestyle.Passcode` style so that the app bar color would be blue rather than purple, which is the default for `Theme.MaterialComponents.Light.NoActionBar`, in the recent apps view.  See the screenshots below for illustration.

![update_passcode_layout_again_phone](https://user-images.githubusercontent.com/3827611/69195618-6f594300-0ae9-11ea-925d-66594abd382d.png)

![update_passcode_layout_again_tablet](https://user-images.githubusercontent.com/3827611/69195517-17bad780-0ae9-11ea-809d-623d58a9bf4d.png)

### Test
The fingerprint icon shown below the 7 is only visible if the device has fingerprint authentication hardware.  If the device does not have that hardware, the icon is not shown.
1. Open navigation drawer.
2. Tap ***Settings*** item in navigation drawer.
3. Notice ***Passcode Lock*** section uses "passcode" rather than "PIN" references.
4. Tap ***Turn passcode lock on*** item under ***Passcode Lock*** section.
5. Enter passcode.
6. Enter passcode again.
7. Notice app layout is as screenshots show above.

### Review
Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release
`RELEASE-NOTES.txt` was updated in 4ddf8a46 with:
> Updated passcode lock design
